metadata-ingestion/src/datahub/sql_parsing/sql_parsing_aggregator.py

```diff
@@ -1200,6 +1201,7 @@ def gen_metadata(self) -> Iterable[MetadataChangeProposalWrapper]:
         queries_generated: Set[QueryId] = set()
 
         yield from self._gen_lineage_mcps(queries_generated)
+        self.report.num_queries_used_in_lineage = len(queries_generated)
```

Comment:
I wonder if there is a reason why you put it here and not above the first yield.
This way it looks a bit odd

Reply:
ahh, now I see, I guess `_gen_lineage_mcps` will update that set